src/decorators/sub-store.ts

```diff
@@ -0,0 +1,17 @@
+import { Reducer } from 'redux';
+
+export interface IFractalStoreOptions {
+  basePathMethodName: string
+  localReducer: Reducer<any>;
+}
+
+export const SubStore = ({
+  basePathMethodName,
+  localReducer
+}: IFractalStoreOptions) =>
+  (constructor: Function): void => {
+    constructor['@angular-redux::fractal-store::options'] = {
```

Comment:
Yeah - I took a look into this but couldn't figure out how to save properties on instances - seems geared to "static" class level annotations - am I missing something?